src/main/plugins/org.dita.pdf2/xsl/fo/map-elements.xsl

```diff
@@ -8,9 +8,11 @@ See the accompanying LICENSE file for applicable license.
 -->
 <xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
   xmlns:fo="http://www.w3.org/1999/XSL/Format"
+  xmlns:dita-ot="http://dita-ot.sourceforge.net/ns/201007/dita-ot"
+  exclude-result-prefixes="dita-ot"
   version="2.0">
 
-  <xsl:template match="*[contains(@class,' map/topicmeta ')]/*[contains(@class,' map/searchtitle ')]"/>
+  <xsl:template match="*[contains(@class,' map/topicmeta ')]/*[dita-ot:matches-searchtitle-class(@class)]" priority="10"/>
```

Comment:
I don't usually like priorities in our code, but this one is added so that `topicmeta/searchtitle` (with either class att) takes priority over the generic `searchtitle` rule that already exists elsewhere.